src/apps/pages/sections/Spacer.astro

```diff
@@ -1,12 +1,18 @@
 ---
```

Comment:
Eek! I don't like have two versions of this same component. Could we add the `.tsx` version to `/src/components` and import it here?